src/plugins/workspace/public/workspace_client.ts

```diff
@@ -314,6 +315,34 @@ export class WorkspaceClient {
     return result;
   }
 
+  /**
+   * copy saved objects among workspaces
+   *
+   * @param {any[]} objects
+   * @param {string} targetWorkspace
+   * @param {boolean} includeReferencesDeep
+   * @returns {Promise<IResponse<any>>} result for this operation
+   */
+  public async copy(
+    objects: any[],
```

Comment:
Try avoid using `any`, it seems the type should be this:

```suggestion
    objects: Array<{id: string, type: string}>,
```